flake8_async/base.py

```diff
@@ -83,7 +83,7 @@ def __iter__(self):
 
     def cmp(self):
         # column may be ignored/modified when autofixing, so sort on that last
-        return self.line, self.code, self.args, self.col
+        return self.line, self.code, self.args, self.message, self.col
```

Comment:
to make the non-equality check in `tests/test_flake8_async.assert_expected_errors` work, since the only thing differing trio/anyio and asyncio in 2xx errors is the message.